crates/holochain_p2p/src/spawn/actor.rs

```diff
@@ -1005,7 +888,10 @@ impl actor::HcP2p for HolochainP2pActor {
         _timeout_ms: Option<u64>,
         _reflect_ops: Option<Vec<DhtOp>>,
     ) -> BoxFut<'_, HolochainP2pResult<()>> {
-        Box::pin(async move { todo!() })
+        Box::pin(async move {
+            tracing::error!("publish is currently a STUB in holochain_p2p--deferring since we'll eventually get stuff on gossip anyways...");
```

Comment:
This is now implemented in K2, we could hook this up here

Reply:
People will get things by gossip. I still think it's too slow for good user experience. If you just gossiped with somebody and they author some data, you might get lucky and see if via somebody else 2 minutes later. If you're unlucky, it might take the full 5 minutes until you next gossip with that agent.

Reply:
Sooo... [this](https://docs.rs/holochain_p2p/latest/holochain_p2p/trait.HolochainP2pSender.html#method.publish) is the current API that holochain is making use of...

I don't know how to work with any of that in kitsune2... it's going to be like five different operations all bundled into the same weird call.

I vote for deferring that work until we get CI passing on the integration branch first.